docs/gitlab-pipelines.md

```diff
@@ -0,0 +1,249 @@
+
+   ![GitLab Variables](assets/gitlab-variables.png)
+   
+When you create the [CI/CD configuration file](#create-the-gitlab-configuration), your Sauce Labs authentication variables appear like so:
+
+```yaml
+variables:
+  SAUCE_USERNAME: ${SAUCE_USERNAME}
+  SAUCE_ACCESS_KEY: ${SAUCE_ACCESS_KEY}
+```
+
+## Create the Saucectl Configuration
+
+Create the `.sauce` directory at the root of your project and add a `config.yaml` file that points [`saucectl`](cli-reference.md) to your existing `tests` directory. 
+
+With the `suites` field you can specify a group of tests as well as the browser `settings` you wish to use.
+
+Below are some examples:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--puppeteer-->
+
+```yaml
+# ./.sauce/puppeteer.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Puppeteer Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-puppeteer-jest-node
+  version: v0.1.8
+sauce:
+  region: us-west-1
+```
+
+<!--playwright-->
+
+```yaml
+# ./.sauce/playwright.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Playwright Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-playwright-jest-node
+  version: v0.1.9
+sauce:
+  region: us-west-1
+```
+
+<!--testcafe-->
+
+```yaml
+# ./.sauce/testcafe.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing TestCafe Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-testcafe-node
+  version: v0.1.7
+sauce:
+  region: us-west-1
+```
+
+<!--cypress-->
+
+```yaml
+# ./.sauce/cypress.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Cypress Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-cypress-mocha-node
+  version: v0.1.11
+sauce:
+  region: us-west-1
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Create the GitLab Configuration
+
+In the root of your project, create a `gitlab-ci.yaml` file. Below are examples in the supported Testrunner Toolkit test frameworks:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--puppeteer-->
+
+```yaml
+image: saucelabs/stt-puppeteer-jest-node:latest
+
+variables:
+  SAUCE_USERNAME: ${SAUCE_USERNAME}
+  SAUCE_ACCESS_KEY: ${SAUCE_ACCESS_KEY}
+
+stages:
+  - test
+
+test puppeteer:
+  tags: [k8s, multi-runner, build, sjc3]
```

Comment:
We should probably remove some those tags, as they are specific to our sauce labs environment :D (same applies to the other examples).